src/Libraries/CoreNodeModels/CreateList.cs

```diff
@@ -14,6 +14,10 @@ namespace CoreNodeModels
     [NodeCategory(BuiltinNodeCategories.CORE_LISTS_CREATE)]
     [IsDesignScriptCompatible]
     [AlsoKnownAs("DSCoreNodesUI.CreateList", "List.Create")]
+    [InPortNames("item0")]
+    [InPortDescriptions(typeof(Resources),"CreateListPortDataIndex0ToolTip")]
+    [OutPortNames("list")]
+    [OutPortDescriptions(typeof(Resources), "CreateListPortDataResultToolTip")]
```

Comment:
These changes are for?

Reply:
they were part of implementing variable input node port data lookup, it won't be too useful in the PR's current state - it can be removed if you want to make this PR more clear -